src/System.Net.HttpListener/tests/GetContextHelper.cs

```diff
@@ -69,15 +69,7 @@ public static class Helpers
 
         public static void WaitForSocketShutdown(Socket socket)
         {
-            if (PlatformDetection.IsWindows || PlatformDetection.IsOSX)
```

Comment:
I didn't see this, but now you have me nervous that I didn't test it properly on the Windows impl... running those tests again now.